src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,16 @@
-export const TodoList = () => {};
+import { TodoWUser } from '../../types/TodoWUser';
+import { TodoInfo } from '../TodoInfo';
+
+export type Props = {
```

Comment:
```suggestion
export interface Props = {
```
why need to export props type from the component?

Reply:
no need, I didn't see it